code/modules/projectiles/guns/ballistic/bow.dm

```diff
@@ -62,5 +62,4 @@
 	desc = "Some sort of pipe made projectile weapon made of a silk string and lots of bending. Used to fire arrows."
 	icon_state = "pipebow"
 	item_state = "pipebow"
-	inaccuracy_modifier = 1.1 //Made of pipe and in a rush
```

Comment:
are you sure...? this will inherit the modifier of 0 from its parent. its kind of designed to not be totally accurate.

Reply:
^

Reply:
![image](https://user-images.githubusercontent.com/2003111/77577123-7a572200-6e93-11ea-9498-5aefcccde8b8.png)
just forwarding it